src/client/atoms/icons/chat-gpt.tsx

```diff
@@ -0,0 +1,10 @@
+import type { SvgIconProps } from "@mui/material/SvgIcon";
+import SvgIcon from "@mui/material/SvgIcon";
+
+export default function ChatGPTIcon(properties: SvgIconProps) {
```

Comment:
Why are we using the ChatGPT Icon here? Isn't this a general purpose assistant that could feature any model?

Reply:
yes, other providers will then have a different icon. 
You can switch models, in a chat. But right now there are only GPT models